src/jobs/heartbeat.ts

```diff
@@ -0,0 +1,29 @@
+import { v1 } from '@datadog/datadog-api-client';
+import * as Sentry from '@sentry/node';
+import moment from 'moment-timezone';
+
+import { DATADOG_API_INSTANCE } from '@/config';
+
+export async function callDatadog(timestamp: number): Promise<void> {
+  const params: v1.EventCreateRequest = {
+    title: 'Infra Hub Update',
+    text: 'Infra Hub is up',
+    alertType: 'error',
+    dateHappened: timestamp,
+    tags: [
+      `source_tool:infra-hub`,
+      `source:infra-hub`,
+      `source_category:infra-tools`,
+      `sentry_user:infra-hub`,
+    ],
+  };
+  await DATADOG_API_INSTANCE.createEvent({ body: params });
+}
+
+export async function heartbeat() {
+  try {
+    await callDatadog(moment().unix());
```

Comment:
Is the main point of the heartbeat to monitor whether are cron jobs are able to be routed? Shouldn't we try to dogfood and make use of Sentry crons for this since that is basically what we are doing here? Also, this doesn't help us detect if a specific endpoint is broken, just that the routing logic (mostly) works, which is still a good thing to monitor, but it isn't a catch all for all problems.

Reply:
The point of the heartbeat is to monitor if the entire system is reachable / available, or if it's down (like what happened last week).